### PR TITLE
Create 2023-01-20-bind9-monitoring.md

### PR DESCRIPTION
**Pull request by shyamvalsan**
BIND 9 Monitoring Use Case Blog

### CODE REVIEW

File: blog/2023-01-20-bind9-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+You should now see the `bind` section on the Overview tab in Netdata Cloud already populated with charts about all the metrics you care about.
+
+Netdata has a public [demo space](https://app.netdata.cloud/spaces/netdata-demo) (no login required) where you can explore different monitoring use-cases and get a feel for Netdata.
+
+## What BIND 9 metrics are important to monitor?
+
+### Clients
+
+**Recursive Clients**
```
Comment (sashwathn) [inline blog/2023-01-20-bind9-monitoring.md:33]: Is this description correct? Aren't these the clients making recursive queries?

**Review by sashwathn — APPROVED** [review]
One minor comment..

File: blog/2023-01-20-bind9-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+Recursive Clients measures the number of clients who are making DNS queries to the BIND 9 DNS server. Normal value ranges for this metric will depend on the type of DNS server, the number of domains it is managing, and the number of clients making requests.
```
Comment (shyamvalsan) [inline blog/2023-01-20-bind9-monitoring.md:35]:
```suggestion
Recursive Clients measures the number of clients who are making recursive DNS queries to the BIND 9 DNS server. Normal value ranges for this metric will depend on the type of DNS server, the number of domains it is managing, and the number of clients making requests.
```

File: blog/2023-01-20-bind9-monitoring.md (hunk 1 above)
Comment (shyamvalsan) [reply blog/2023-01-20-bind9-monitoring.md:33]: updated

**Pull request merged**